static/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+project(pyArborX_static LANGUAGES CXX)
+
+#choose modules that should be included in the build
+set(PY_STATIC_MODULES Point Box)
+
+# get source files for selected static modules
+foreach(PY_MODULE IN LISTS PY_STATIC_MODULES)
+  list(APPEND PY_SRC ${PROJECT_SOURCE_DIR}/src/pyArborX_${PY_MODULE}.cpp)
+  list(APPEND MODULE_INCLUDES "\n#include \"pyArborX_${PY_MODULE}.hpp\"" )
+  list(APPEND MODULE_GENERATORS "\n generate${PY_MODULE}Wrapper(m)")
+endforeach()
```

Comment:
I did it in this way so adding a new class to the bindings only requires the source and header file and adding it to the `PY_STATIC_MODULES` here. Otherwise we can add the includes and calls direct to the `pyArborX_static.cpp`.

The current solution allows to automatically generate bindings for `Kokkos::View<ArborX::Geometry>` without having to repeat the implementation of the bindings for `View`.